chore: Removed semantic release changelog functionality

jansentjeu opened the pull request.

edwinljacobs commented:
Hi thanks for the pull request, I already made a branch (see commit: https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/8996fc8489d502a4ee2b2618594007f3932a236a) to fix this issue. The decorator itself has a method getTableName which does the same as in this pull request.

see https://github.com/EmicoEcommerce/Magento2TweakwiseExport/blob/master/src/Model/Write/Products/CollectionDecorator/AbstractDecorator.php

I was planning on merging this today.

govereem commented:
Okay nice,  your solution looks nicer for changing $connection->getTableName() to $ths->getTableName(). My solution can be skipped.

ahuininga-orisha committed:
chore: manually trigger release

ahuininga-orisha committed:
Trigger release manually (#56)

* chore: manually trigger release

* Update release.yml

semantic-release-bot committed:
chore(release): 7.0.0-beta.1 [skip ci]

# [7.0.0-beta.1](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/compare/v6.0.2...v7.0.0-beta.1) (2024-04-12)

### Bug Fixes

* Applied phpcs and phpmd rules [#38](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/38) ([e3966fd](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/e3966fd147d3d0816f20a4671facd0180ccd949d))
* Trigger release ([7334c1e](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/7334c1e74a79ed688fa640b6b2d3fe4b637162eb))
* use id instead of SKU [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) ([f0070ec](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/f0070ec08f8f7d3dd5d9af47a7fd14cdf1b0a416))

* Merge beta into master (#45) ([12701f7](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/12701f78e66cedf7f932ebae56fd973005e8a6c4)), closes [#45](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/45) [#30](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/30) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#37](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/37) [#36](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/36)

### BREAKING CHANGES

* Code is refactored based on phpcs and phpmd rules.
* Code is refactored based on phpcs and phpmd rules.
* Code is refactored based on phpcs and phpmd rules.

* fix: Fixed DriverInterface issue

* chore(release): 7.0.0-beta.1 [skip ci]

# [7.0.0-beta.1](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/compare/v6.0.2...v7.0.0-beta.1) (2024-03-29)

### Bug Fixes

* Fixed DriverInterface issue ([a6d0109](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/a6d01098c246b2a3b3475ed0d3060d4de75f2817))
* Code is refactored based on phpcs and phpmd rules. #37 #36

ahuininga-orisha committed:
Merge branch 'master' into beta

semantic-release-bot committed:
chore(release): 7.0.0-beta.1 [skip ci]

# [7.0.0-beta.1](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/compare/v6.0.2...v7.0.0-beta.1) (2024-04-12)

### Bug Fixes

* Applied phpcs and phpmd rules [#38](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/38) ([e3966fd](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/e3966fd147d3d0816f20a4671facd0180ccd949d))
* Trigger release ([7334c1e](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/7334c1e74a79ed688fa640b6b2d3fe4b637162eb))
* use id instead of SKU [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) ([f0070ec](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/f0070ec08f8f7d3dd5d9af47a7fd14cdf1b0a416))

* Merge beta into master (#45) ([12701f7](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/12701f78e66cedf7f932ebae56fd973005e8a6c4)), closes [#45](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/45) [#30](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/30) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#39](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/39) [#37](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/37) [#36](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/issues/36)

### BREAKING CHANGES

* Code is refactored based on phpcs and phpmd rules.
* Code is refactored based on phpcs and phpmd rules.
* Code is refactored based on phpcs and phpmd rules.

* fix: Fixed DriverInterface issue

* chore(release): 7.0.0-beta.1 [skip ci]

# [7.0.0-beta.1](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/compare/v6.0.2...v7.0.0-beta.1) (2024-03-29)

### Bug Fixes

* Fixed DriverInterface issue ([a6d0109](https://github.com/EmicoEcommerce/Magento2TweakwiseExport/commit/a6d01098c246b2a3b3475ed0d3060d4de75f2817))
* Code is refactored based on phpcs and phpmd rules. #37 #36

jansentjeu committed:
chore: Removed semantic release changelog functionality

jansentjeu committed:
Merge remote-tracking branch 'origin/master' into beta

# Conflicts:
#	.github/workflows/release.yml
#	CHANGELOG.md
